### DISCUSSION TITLE
Alter the image onscreen using a slider or keypress

### DISCUSSION THREAD

**Discussion by remarknibor** [jspsych/jsPsych#2167] [General]
If I had 10 versions of a single face, let's say, varying from more to less masculine, and I wanted the participant to cycle through these and pick one, how could this be done?
They'd start with one (random) version onscreen, and then using a slider or perhaps the arrow keys, they'd cycle through these images (the one onscreen would be replaced each time with the one before or after in the array of images, depending on what the user did, so it looked like they were altering it) and then select one version as their response (via a key press or button click).
Which plugins would be useful here? Where would I begin?
Thanks in advance.

**Comment by jodeleeuw** [comment]
The reconstruction plugin would work for this. You would just need to write the function so that it stepped through an array of images and displayed the appropriate one.
